packages/netlify-cms-widget-markdown/src/serializers/__tests__/remarkShortcodes.spec.js

```diff
@@ -22,34 +23,46 @@ describe('remarkParseShortcodes', () => {
   describe('pattern matching', () => {
     it('should work', () => {
       const editorComponent = EditorComponent({ pattern: /bar/ });
-      process('foo bar', [editorComponent]);
+      process('foo bar', Map([[editorComponent.id, editorComponent]]));
```

Comment:
Will using object notation be simpler? e.g. `Map({ [editorComponent.id]: editorComponent })`

Reply:
Sure!